.github/workflows/apptek_hashes.yml

```diff
@@ -0,0 +1,49 @@
+name: AppTek hashes
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+jobs:
+  test-hashes:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/setup-python@v4
+      with:
+        python-version: 3.8
+    - name: Start Bitbucket Pipeline
+      run: |
+        curl -X POST -s -u ${{ secrets.MICHEL_BITBUCKET_AUTHENTICATION }} \
+            -H 'Content-Type: application/json' \
+            https://api.bitbucket.org/2.0/repositories/omnifluent/apptek_asr/pipelines/ \
+            -d '{"target": {"ref_type": "branch", "ref_name": "willi-i6_core-integration",
+                 "type": "pipeline_ref_target", "selector": {"type": "custom", "pattern": "hashes-only"}},
+                 "variables": [{"key": "i6_core_branch", "value": "'${GITHUB_HEAD_REF}'"}]}' |\
+        python -c "import sys,json; print(json.load(sys.stdin)['uuid'])" |\
+        sed 's/{/%7B/' | sed 's/}/%7D/' > pipeline_uuid.txt
+    - name: Wait for Results
+      run: |
+        while [ COMPLETED != $(\
+            curl -X GET -s -u ${{ secrets.MICHEL_BITBUCKET_AUTHENTICATION }} \
```

Comment:
there is a lot of repetition in the curl commands, could we put the flags that are always used in env variables (e.g. $CURL_FLAGS) and reference them here?

Reply:
I have tried to unify it in a sensible way now.